sources/packages/api/src/services/student/student.service.ts

```diff
@@ -176,6 +176,20 @@ export class StudentService extends RecordDataModelService<Student> {
     return this.repo.find({ validSIN: null });
   }
 
+  /**
+   * Gets a student's valid sin status
+   * @returns Students SIN validation status.
+   */
+  async getStudentSinStatus(userId: number): Promise<boolean | null> {
+    const sinStatus = await this.repo
+      .createQueryBuilder("student")
+      .leftJoinAndSelect("student.user", "user")
+      .where("student.userId = :userIdParam", { userIdParam: userId })
+      .select("student.validSin")
```

Comment:
<img width="284" alt="image" src="https://user-images.githubusercontent.com/62901416/149427226-fe33f8ba-deac-4721-8167-8745001eba34.png">
 Can we please change the validSin to "student.validSIN" as part of db model?